test/web/10-VerifiableCredentialStore.js

```diff
@@ -125,27 +129,30 @@ describe('VerifiableCredentialStore', () => {
     const newCred = Object.assign({}, AlumniCredential, {id: 'foo'});
     await vcStore.insert({credential: newCred});
 
-    const credentials = await vcStore.match({query: query.query1});
+    // query with an issuer that is not the same as the issuer on the credential
+    const credentials = await vcStore.match({
+      query: queryWithIncorrectTrustedIssuer
```

Comment:
`Incorrect` is confusing -- how can an issuer be "incorrect"? Perhaps `queryWithMatchingTrustedIssuer` and `queryWithNonMatchingTrustedIssuer`?

Reply:
Addressed here f0cd1d0